### PR TITLE
JBIDE-27320 Remove old JRE from server parameters(4, 5, 6 and 7)

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

### CODE REVIEW

**Comment by jkopriva** [comment]
Several builds: https://studio-jenkins-csb-codeready.cloud.paas.psi.redhat.com/job/Studio/job/Quality/job/Integration-tests/job/archives-itests/
https://studio-jenkins-csb-codeready.cloud.paas.psi.redhat.com/job/Studio/job/Quality/job/Integration-tests/job/server-management-itests/14/
https://studio-jenkins-csb-codeready.cloud.paas.psi.redhat.com/job/Studio/job/Quality/job/Integration-tests/job/server-as-itests/14/
https://studio-jenkins-csb-codeready.cloud.paas.psi.redhat.com/job/Studio/job/Quality/job/Integration-tests/job/server-itests/26/

I have tested Tomcat ITests just locally and it went fine.

**Comment by jkopriva** [comment]
image

File: wtp/itests/org.jboss.tools.wtp.runtimes.tomcat.itests/src/org/jboss/tools/wtp/runtimes/tomcat/itests/AbstractTomcatDetectionTest.java [hunk 1]
```diff
@@ -121,21 +121,21 @@ protected String toString(Object[] runtimes) {
...
-      IExecutionEnvironment se7Env = EnvironmentsManager.getDefault().getEnvironment(IExecutionEnvironmentConstants.EXEC_ENV_JavaSE17);
-      System.out.println("jre7 home is " + JRE_7_HOME);
-      if (JRE_7_HOME != null && !JRE_7_HOME.startsWith("${")) {
-        assertTrue("JRE7 home " + JRE_7_HOME + " does not exist", new File(JRE_7_HOME).exists());
-        IVMInstall foundOrCreated = JREUtils.findOrCreateJRE(new Path(JRE_7_HOME));
+	private static void checkJRE8Availability() {
+      IExecutionEnvironment se8Env = EnvironmentsManager.getDefault().getEnvironment(IExecutionEnvironmentConstants.EXEC_ENV_JavaSE18);
+      System.out.println("jre8 home is " + JRE_8_HOME);
+      if (JRE_8_HOME != null && !JRE_8_HOME.startsWith("${")) {
+        assertTrue("JRE7 home " + JRE_8_HOME + " does not exist", new File(JRE_8_HOME).exists());
```
Comment (sbouchet) [inline wtp/itests/org.jboss.tools.wtp.runtimes.tomcat.itests/src/org/jboss/tools/wtp/runtimes/tomcat/itests/AbstractTomcatDetectionTest.java:128]: please fix typo here
```suggestion
        assertTrue("JRE8 home " + JRE_8_HOME + " does not exist", new File(JRE_8_HOME).exists());
```

Comment (jkopriva) [reply wtp/itests/org.jboss.tools.wtp.runtimes.tomcat.itests/src/org/jboss/tools/wtp/runtimes/tomcat/itests/AbstractTomcatDetectionTest.java]: Thanks! Fixed.

**Pull request merged**